src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -1250,9 +1275,6 @@ private void BuildChunksCache()
                 // Ignore chunks that wouldn't yield any useful compounds when absorbing
                 ref var compounds = ref chunk.Get<CompoundStorage>();
 
-                if (!compounds.Compounds.HasAnyCompounds())
```

Comment:
Is it intentional that AI considers chunks that don't have compounds for eating? I foresee this causing issues like in the ice biome cells trying to eat the damaging chunks.

Reply:
I must've consider big iron chunks not compound filled or something.

Reply:
Well at least with the reverted biomes.json, this change shouldn't be necessary.